platform/marvell-armhf/docker-syncd-mrvl/Dockerfile.j2

```diff
@@ -26,7 +26,7 @@ RUN dpkg -i \
 debs/{{ deb }}{{' '}}
 {%- endfor %}
 
-COPY ["start.sh", "syncd.sh", "/usr/bin/"]
+COPY ["start.sh", "/usr/bin/"]
```

Comment:
why remove syncd.sh?

Reply:
For Marvell right now there is no need to override common syncd.sh with platform specific syncd.sh.